ex/loqui/lib/loqui/client.ex

```diff
@@ -76,8 +160,8 @@ defmodule Loqui.Client do
 
   @default_timeout 5000
   @max_sequence round(:math.pow(2, 32) - 1)
-  @default_compressors %{Compressors.NoOp.name() => Compressors.NoOp}
-  @default_codecs %{Codecs.Erlpack.name() => Codecs.Erlpack}
+  @default_compressors Enum.into(Compressors.all(), %{}, &{&1.name, &1})
```

Comment:
Interesting that this is `.name` but was `.name()` in the previous code. Is it something about anonymous functions and macros?